tests/common/quantization/metatypes.py

```diff
@@ -107,8 +107,39 @@ class AddTestMetatype(TestMetatype):
     name = 'add'
 
 
+@METATYPES_FOR_TEST.register()
```

Comment:
Why should we keep a copy of each metatype separate for tests? Why not use backend-specific metatypes directly? 
It will help us to avoid such misses while adding the new metatypes.

Reply:
It's our design. These metatypes had been here before my changes. We need the common metatypes to test the common code. By the way, QPS algorithm is tested using these metatypes